.github/CODEOWNERS

```diff
@@ -44,10 +44,10 @@
 /docs/visual-basic/ @KathleenDollard @BillWagner
 
 # The ML.NET Guide:
-/docs/machine-learning/ @luisquintanilla
+/docs/machine-learning/ @luisquintanilla @JakeRadMSFT @michaelgsharp
 
 # The .NET for Apache Spark Guide:
-/docs/spark/ @luisquintanilla
+/docs/spark/ @luisquintanilla  @JakeRadMSFT @michaelgsharp
```

Comment:
This folder doesn't exist anymore. Also perhaps you should add me as a reviewer for ML.NET.